python3Packages.oslo-concurrency: 7.1.0 -> 7.2.0

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from passthru.updateScript.


meta.description for python3Packages.oslo-concurrency is: Oslo Concurrency library

meta.homepage for python3Packages.oslo-concurrency is: https://github.com/openstack/oslo.concurrency




###### Updates performed
- Ran passthru.UpdateScript

###### To inspect upstream changes





###### Impact

<b>Checks done</b>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 7.2.0 with grep in /nix/store/d82xcarphrk7mr3wmpjffargnrpsvkmz-python3.13-oslo-concurrency-7.2.0
- found 7.2.0 in filename of file in /nix/store/d82xcarphrk7mr3wmpjffargnrpsvkmz-python3.13-oslo-concurrency-7.2.0

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
25 total rebuild path(s)

25 package rebuild(s)

First fifty rebuilds by attrpath

magnumclient
mistralclient
openstackclient-full
python312Packages.oslo-concurrency
python312Packages.osprofiler
python312Packages.python-aodhclient
python312Packages.python-magnumclient
python312Packages.python-mistralclient
python312Packages.python-neutronclient
python312Packages.python-octaviaclient
python312Packages.python-troveclient
python312Packages.subunit2sql
python312Packages.tempest
python313Packages.oslo-concurrency
python313Packages.osprofiler
python313Packages.python-aodhclient
python313Packages.python-magnumclient
python313Packages.python-mistralclient
python313Packages.python-neutronclient
python313Packages.python-octaviaclient
python313Packages.python-troveclient
python313Packages.subunit2sql
python313Packages.tempest
troveclient
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from the cache**:
```
nix-store -r /nix/store/d82xcarphrk7mr3wmpjffargnrpsvkmz-python3.13-oslo-concurrency-7.2.0 \
  --option binary-caches 'https://cache.nixos.org/ https://nixpkgs-update-cache.nix-community.org/' \
  --option trusted-public-keys '
  nixpkgs-update-cache.nix-community.org-1:U8d6wiQecHUPJFSqHN9GSSmNkmdiFW7GW7WNAnHW0SM=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The nixpkgs-update cache is only trusted for this store-path realization.)
For the cached download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A python3Packages.oslo-concurrency https://github.com/r-ryantm/nixpkgs/archive/7bd24a20ef0e7b614c3318babc03a8b67460b36b.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/7bd24a20ef0e7b614c3318babc03a8b67460b36b#python3Packages.oslo-concurrency
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/d82xcarphrk7mr3wmpjffargnrpsvkmz-python3.13-oslo-concurrency-7.2.0
ls -la /nix/store/d82xcarphrk7mr3wmpjffargnrpsvkmz-python3.13-oslo-concurrency-7.2.0/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>
  <ul>
    <li>python312Packages.subunit2sql</li>
    <li>python312Packages.subunit2sql.dist</li>
    <li>python313Packages.subunit2sql</li>
    <li>python313Packages.subunit2sql.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 38 packages built:</summary>
  <ul>
    <li>magnumclient (python313Packages.python-magnumclient)</li>
    <li>magnumclient.dist (python313Packages.python-magnumclient.dist)</li>
    <li>mistralclient (python313Packages.python-mistralclient)</li>
    <li>mistralclient.dist (python313Packages.python-mistralclient.dist)</li>
    <li>openstackclient-full</li>
    <li>openstackclient-full.dist</li>
    <li>python312Packages.oslo-concurrency</li>
    <li>python312Packages.oslo-concurrency.dist</li>
    <li>python312Packages.osprofiler</li>
    <li>python312Packages.osprofiler.dist</li>
    <li>python312Packages.python-aodhclient</li>
    <li>python312Packages.python-aodhclient.dist</li>
    <li>python312Packages.python-magnumclient</li>
    <li>python312Packages.python-magnumclient.dist</li>
    <li>python312Packages.python-mistralclient</li>
    <li>python312Packages.python-mistralclient.dist</li>
    <li>python312Packages.python-neutronclient</li>
    <li>python312Packages.python-neutronclient.dist</li>
    <li>python312Packages.python-octaviaclient</li>
    <li>python312Packages.python-octaviaclient.dist</li>
    <li>python312Packages.python-troveclient</li>
    <li>python312Packages.python-troveclient.dist</li>
    <li>python312Packages.tempest</li>
    <li>python312Packages.tempest.dist</li>
    <li>python313Packages.oslo-concurrency</li>
    <li>python313Packages.oslo-concurrency.dist</li>
    <li>python313Packages.osprofiler</li>
    <li>python313Packages.osprofiler.dist</li>
    <li>python313Packages.python-aodhclient</li>
    <li>python313Packages.python-aodhclient.dist</li>
    <li>python313Packages.python-neutronclient</li>
    <li>python313Packages.python-neutronclient.dist</li>
    <li>python313Packages.python-octaviaclient</li>
    <li>python313Packages.python-octaviaclient.dist</li>
    <li>troveclient (python313Packages.python-troveclient)</li>
    <li>troveclient.dist (python313Packages.python-troveclient.dist)</li>
    <li>python313Packages.tempest</li>
    <li>python313Packages.tempest.dist</li>
  </ul>
</details>

---

###### Maintainer pings

cc @SuperSandro2000 @anthonyroussel @vinetos for [testing](https://github.com/ryantm/nixpkgs-update/blob/main/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

> [!TIP]
> As a maintainer, if your package is located under `pkgs/by-name/*`, you can comment **`@NixOS/nixpkgs-merge-bot merge`** to automatically merge this update using the [`nixpkgs-merge-bot`](https://github.com/NixOS/nixpkgs-merge-bot).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

r-ryantm committed:
python3Packages.oslo-concurrency: 7.1.0 -> 7.2.0

SuperSandro2000 commented on pkgs/development/python-modules/oslo-concurrency/default.nix:
```diff
@@ -19,13 +19,13 @@
+  version = "7.2.0";
   pyproject = true;
 
   src = fetchPypi {
```
Let's change that to GitHub

SuperSandro2000 committed:
python313Packages.oslo-concurrency: fetch from GitHub

SuperSandro2000 committed:
python313Packages.oslo-concurrency: enable tests again, disable hanging tests

SuperSandro2000 committed:
python313Packages.oslo-concurrency: fix tests on darwin

SuperSandro2000 committed:
python313Packages.osc-lib: fix tests on darwin

SuperSandro2000 committed:
python313Packages.python-octaviaclient: do not mix python versions

openstackclient is the wrapped package which exposes the cli. For the
python package we want to use python-openstackclient.
